MAX_ACCELERATION: 1.0
MAX_YAWRATE: 0.8
MAX_D_YAWRATE: 5.0
MAX_WHEEL_ANGULAR_VELOCITY: 11.6
WHEEL_RADIUS: 0.125
TREAD: 0.5
